releases/release-6.1.3.md

```diff
@@ -0,0 +1,92 @@
+---
+title: TiDB 6.1.3 Release Notes
+---
+
+# TiDB 6.1.3 Release Notes
+
+Release date: xx, xx, 2022
+
+TiDB version: 6.1.3
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.1/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v6.1.3#version-list)
+
+## Improvements
+
```

Comment:
- Enable transaction split and disable safeMode of a changefeed in TiCDC by defult. [#7505](https://github.com/pingcap/tiflow/issues/7505)
 - Improve kakfa protocol encoder performance.[#7540](https://github.com/pingcap/tiflow/issues/7540)[#7532](https://github.com/pingcap/tiflow/issues/7532)[#7543](https://github.com/pingcap/tiflow/issues/7532)